### PR TITLE
Refactoring StorPool's smoke tests

### PR DESCRIPTION
**Pull request by slavkap**
### Description

- removed the hardcoded endpoint of the StorPool primary storage from tests. It should be provided in the configuration file
- added the git commit information into the maven build. The information is used for monitoring

### Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

#### Feature/Enhancement Scale

- [ ] Major
- [X] Minor

### CODE REVIEW

File: plugins/storage/volume/storpool/pom.xml [hunk 1]
```diff
@@ -78,6 +83,35 @@
...
+					<excludeProperties>
+						<excludeProperty>git.*.email</excludeProperty>
+					</excludeProperties>
+					<gitDescribe>
+						<skip>false</skip>
+						<always>false</always>
+						<dirty>-dirty</dirty>
+					</gitDescribe>
+				</configuration>
+			</plugin>
```
Comment (DaanHoogland) [inline plugins/storage/volume/storpool/pom.xml:114]: tabs instead of spaces?

**Comment by DaanHoogland** [comment]
@slavkap , a lot of failures of which it is unclear what caused it,
I'll run smoke tests.

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Review by rohityadavcloud — APPROVED** [review]

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by rohityadavcloud** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by weizhouapache** [comment]
@slavkap 
could you please fix the build errors ?

**Comment by DaanHoogland** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test matrix

**Comment by DaanHoogland** [comment]
@blueorangutan test rocky8 vmware-67u3

**Review by weizhouapache — APPROVED** [review]
code lgtm

@slavkap this is ready for merge.
have you verified this test on your local environment ? just to confirm

**Comment by slavkap** [comment]
Hi @weizhouapache, yes, the changes have been tested in our lab.

**Comment by weizhouapache** [comment]
> Hi @weizhouapache, yes, the changes have been tested in our lab.

Thanks @slavkap 
Merging

**Pull request merged**